src/index.ts

```diff
@@ -173,10 +94,37 @@ function getAsyncOperationId(message) {
  * @param {string} value - value to set variable to if not undefined
  * @returns {undefined} - none
  */
-function safeSetEnv(VARNAME, value) {
+function safeSetEnv(VARNAME: string, value: string) : void {
   if (value !== undefined) process.env[VARNAME] = value;
 }
 
+// eslint-disable-next-line require-jsdoc
+function iscumulusMessageWithGranulesInPayload(
```

Comment:
```suggestion
function isCumulusMessageWithGranulesInPayload(
```

Reply:
6fbfde6, as it needed updated both places.